lib/util/estraverse.js

```diff
@@ -33,6 +33,7 @@ var experimentalKeys = {
  */
 function installKeys(keys) {
     for (var key in keys) {
+        /* istanbul ignore else */
```

Comment:
it just a wrapper around the library, it get executed at load time so its difficult to test and I think not worth the effort to add tests. (Also we dont have any individual test for this file)